engine/src/main/scala/cromwell/engine/workflow/lifecycle/WorkflowFinalizationActor.scala

```diff
@@ -57,6 +63,13 @@ case class WorkflowFinalizationActor(workflowIdForLogging: WorkflowId, workflowD
   override def successResponse(data: WorkflowLifecycleActorData) = WorkflowFinalizationSucceededResponse
   override def failureResponse(reasons: Seq[Throwable]) = WorkflowFinalizationFailedResponse(reasons)
 
+  // If a finalization actor dies, send ourselves the failure and stop the child actor
+  override def supervisorStrategy = OneForOneStrategy() {
```

Comment:
Either the comment is wrong or this supervision should be in the `Finalization` actor's supervisor, not the actor itself